src/battle.cpp

```diff
@@ -0,0 +1,74 @@
+#include "battle.h"
+#include <cstdlib>
+#include <ctime>
+#include <sstream>
+
+Battle::Battle(){
+  m = new Model("./lyrics/lyrics.txt");
+
+  std::cout<<"gimme a bar\n";
+  std::getline(std::cin, given);
+  findLastAndCount();
+}
+
+std::string Battle::traceBack(){
+  //find a word that rhymes with the "last" in Model
+  //outLastfindRhyme(last)
```

Comment:
I think generally we should try to avoid committing commented out code for readability's sake.